docs/src/test-components-js.md

```diff
@@ -0,0 +1,101 @@
+---
+id: test-components
+title: "Experimental: Components"
+---
+
+Playwright Test can now test your components.
+
+<!-- TOC -->
+
+## Example
+
+Here is what a typical component test looks like:
+
+```js
+test('event should work', async ({ mount }) => {
+  let clicked = false;
+
+  // Mount a component. Returns locator pointing to the component.
+  const c = await mount(<Button title='Submit'
+    onClick={() => clicked = true}>
+  </Button>);
+
+  // As with any Playwright test, assert locator text.
+  await expect(component).toContainText('Submit');
+
+  // Perform locator click. This will trigger the event.
+  await component.click();
+
+  // Assert that respective events have been fired.
+  expect(clicked).toBeTruthy();
+})
+```
+
+## How to get started
+
+Adding Playwright Test to an existing React, Vue or Svelte project is easy. Below at the steps to enable Playwright Test for a sample create-react-app with TypeScript template.
+
+### Step 1: Install Playwright Test for components for your respective framework
+
+```sh
+npm i @playwright/test
+npm i @playwright/experimental-ct-react
+# npm i @playwright/experimental-ct-vue
+# npm i @playwright/experimental-ct-svelte
+```
+
+### Step 2: create `playwright/index.html`
+```html
+<html lang="en">
+  <body>
+    <div id="root"></div>
```

Comment:
```suggestion
```

Reply:
They should know they can put it into a container for testing